src/navigator/NavigationService.ts

```diff
@@ -225,25 +225,17 @@ export async function isBottomSheetVisible(screen: Screens) {
   return !!state?.routes.find((route: any) => route.name === screen)
 }
 
-interface NavigateHomeOptions {
-  params?: StackParamList[Screens.DrawerNavigator] | StackParamList[Screens.TabNavigator]
-}
-
 /***
  * Navigates to the home screen resetting the navigation stack by default
  * If called from a modal make sure to pass fromModal: true. Otherwise it will cause a null pointer dereference and subsequent app crash
  * TODO: stop using ReactNative modals and switch to react-navigation modals
  */
-export function navigateHome(options?: NavigateHomeOptions) {
-  const { params } = options ?? {}
-  const timeout = params?.fromModal && Platform.OS === 'ios' ? 500 : 0
+export function navigateHome(fromModal?: boolean) {
```

Comment:
Made this more single purpose as navigating to any drawer / tab screen would mean callers have to conditionally pass the right screen names, and also can't use this for navigating screens only in the drawer (e.g., Settings)